src/modules/commentPopup.js

```diff
@@ -122,15 +123,15 @@ const showCommentPopup = (movieId) => {
   closeCommentPopup();
 };
 
-const displayComments = () => {
-  const commentBtns = document.querySelectorAll('.comment-btn');
-  commentBtns.forEach((movie) => {
-    movie.addEventListener('click', () => {
-      const movieId = movie.getAttribute('movie-id');
-      showCommentPopup(movieId);
-      document.body.style.overflow = 'hidden';
-    });
-  });
-};
+// const displayComments = () => {
+//   const commentBtns = document.querySelectorAll('.comment-btn');
+//   commentBtns.forEach((movie) => {
+//     movie.addEventListener('click', () => {
+//       const movieId = movie.getAttribute('movie-id');
+//       showCommentPopup(movieId);
+//       document.body.style.overflow = 'hidden';
+//     });
+//   });
+// };
```

Comment:
- [ ] Please kindly remove `commented code` from your `code base`, as it make your `code base` dirty and less readable.